src/modules/new-request-form/fields/helpers.test.ts

```diff
@@ -0,0 +1,101 @@
+import type { Field, Filter } from "../data-types";
```

Comment:
It's a good thing you added unit tests. Unfortunately, we don't have that many right now, but in general, the convention is to have the test file named `[SOURCE_FILE].test.ts`. The name `helpers.test.ts` doesn't make it clear where the implementation leaves.